fe1-web/model/network/method/message/data/MessageData.ts

```diff
@@ -28,6 +28,7 @@ export enum ActionType {
   RESULT = 'result',
   CLOSE = 'close',
   ADD = 'add',
+  ADD_BROADCAST = 'addBroadcast',
```

Comment:
I thought the same, but actually this is what have been decided in the protocol. Maybe we can talk about it next meeting. Anyway, this can be easily changed but the backend needs to follow.

Reply:
Yeah I'm confused too, it's probably wort to have a chat about this whenever it's convenient :)

Reply:
I somehow feel that this is really a pointer to a chirp, notifying about the existence of a chirp.
Something like chirp/notify might be more obvious.